gulpfile.js

```diff
@@ -274,6 +296,60 @@ gulp.task('minifyRelease', ['generateStubs'], function() {
 
 gulp.task('release', ['combine', 'minifyRelease', 'generateDocumentation']);
 
+gulp.task('test', ['build'], function(done) {
+    var argv = yargs.argv;
+
+    var enableAllBrowsers = false;
+    var includeCategory = '';
+    var excludeCategory = '';
+    var webglValidation = false;
+
+    if (argv.all) {
+        enableAllBrowsers = true;
+    }
+
+    if (argv.include) {
+        includeCategory = argv.include;
+    }
+
+    if (argv.exclude) {
+        excludeCategory = argv.exclude;
+    }
+
+    if (argv.webglValidation) {
+        webglValidation = true;
+    }
+
+    test(enableAllBrowsers, includeCategory, excludeCategory, webglValidation, false, done);
+});
+
+gulp.task('test-release', ['minifyRelease'], function(done){
```

Comment:
Sorry for not realizing this until now, but it just occurred to me that we don't want to have and of the `test` tasks depend on anything.  The problem is that we often want to run the tests without building (for example, using the already produced release zip).  This means you could probably go back to a single test task that takes parameters, but that's up to you, whichever you think is cleaner.  Sorry again for the confusion.